lib/Extractor/ExprBuilder.cpp

```diff
@@ -376,6 +376,22 @@ Inst *ExprBuilder::getDemandedBitsCondition(Inst *I) {
                                    LIC->getInst(Inst::Eq, 1, {Res, Zero})});
     Result = LIC->getInst(Inst::And, 1, {Result, SignBits});
   }
+  if (!I->Range.isEmptySet() && !I->Range.isFullSet()) {
```

Comment:
please add a TODO saying that we might want to look into handling the EmptySet case later